README.md

```diff
@@ -145,6 +156,174 @@ readData(dataInput) {
 }
 ```
 
+# Setting Up Client Network
+
+### 7. Enabling Client TLS/SSL
+
```

Comment:
We are discouraging Hazelcast users to use SSL (the Java ones). I don't know if it also applies to the connections between Node.js and clusters. Just noticing. @mustafaiman ?

Reply:
I was not aware of such discouragement. I will check with the team.